packages/core/src/Stores/cfd-account-store.js

```diff
@@ -0,0 +1,31 @@
+import { CFD_PLATFORMS, getAccountListKey } from '@deriv/shared';
```

Comment:
just a clarification: we will not use TS for store?

Reply:
we haven't started TS migration on stores yet, I followed the same approach with current stores.